content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -134,3 +134,90 @@ updateUser: (parent, args, ctx: Context) => {
   })
 },
 ```
+
+## The effect of <inlinecode>null</inlinecode> and <inlinecode>undefined</inlinecode> on conditionals
+
+There are some caveats to filtering with conditionals which might produce unexpected results. When filtering with conditionals you might expect one result but receive another given how Prisma treats nullable values.
+
+### A high level overview
+
+
+|Operator   | 0 filters              | 1 filter               | n filters            |
+|-----------|------------------------|------------------------|----------------------|
+|`OR`       | return empty list      | validate single filter | validate all filters |
+|`AND`      | return all items       | validate single filter | validate all filters |
+|`NOT`      | return all items       | validate single filter | validate all filters |
+
+The following example shows how an `undefined` parameter can impact the expected return data when passed to a conditional operator such as [`OR`](../../../reference/api-reference/prisma-client-reference#or) <span class="api"></span>.
+
+```ts
+interface FormData {
+  name: string;
+  email?: string;
+}
+
+const formData: FormData = {
+  name: 'Emelie'
+}
+
+const users = await prisma.user.findMany({
+  where: {
+    OR: [
+      {
+        email: {
+          contains: formData.email
+        }
+      }
+    ]
+  }
+})
+
+// returns: []
+```
+
+The query is passed the form data which can optionally contain an email. In this instance the email field has been omitted from the form. When this query is run no data is returned.
```

Comment:
Active voice here - 

> The query is passed the form data which can optionally contain an email.

"the query receives filters from a `formData` object, which includes an optional `email` property".

> In this instance the email field has been omitted from the form.

"In this instance, the form does not include an `email`." or
"In this instance, the value of the `email` property is `undefined`"